server/routes/reviewPlan/exemptionReasonController.ts

```diff
@@ -8,33 +9,49 @@ export default class ExemptionReasonController {
   getExemptionReasonView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     const { prisonerSummary } = res.locals
-    const prisonerContext = getPrisonerContext(req.session, prisonNumber)
 
-    const { reviewExemptionForm } = prisonerContext
+    let reviewExemptionForm: ReviewExemptionForm
+    if (getPrisonerContext(req.session, prisonNumber).reviewExemptionForm) {
+      reviewExemptionForm = getPrisonerContext(req.session, prisonNumber).reviewExemptionForm
+    } else {
+      reviewExemptionForm = getPrisonerContext(req.session, prisonNumber).reviewExemptionDto
```

Comment:
Almost ... but not quite. Have a look at what we do in `whoCompletedReviewController`, but its basically this:
```
    let whoCompletedReviewForm: WhoCompletedReviewForm
    if (getPrisonerContext(req.session, prisonNumber).whoCompletedReviewForm) {
      whoCompletedReviewForm = getPrisonerContext(req.session, prisonNumber).whoCompletedReviewForm
    } else {
      whoCompletedReviewForm = toWhoCompletedReviewForm(getPrisonerContext(req.session, prisonNumber).reviewPlanDto)
    }
```

You've done almost the right thing .... but on line 17 here where you have:
```
reviewExemptionForm = getPrisonerContext(req.session, prisonNumber).reviewExemptionDto
```
we actually need a small helper function to create the form from the DTO
And that helper function needs to set the correct value in the `Record` based form field, based on the value of `exemptionReason` (I've not explained that very well!)